app/gui2/src/components/GraphEditor/GraphVisualization.vue

```diff
@@ -200,8 +208,11 @@ watchEffect(async () => {
 })
 
 const isBelowToolbar = ref(false)
-let width = ref<number | null>(null)
+let width = ref<Opt<number>>(props.width)
 let height = ref(150)
+// We want to debounce width changes, because they are saved to the metadata.
+const debouncedWidth = debouncedGetter(() => width.value, 300)
+watch(debouncedWidth, (value) => value != null && emit('update:width', value))
```

Comment:
I think it's not quite working properly. We have `prop.width`, `width` and `debouncedWith` - the `prop.width` will be updated only on debounce, but even then `width` is not updated (ref is only initialized with it).

I think the getter below should return `props.width`, and we could have a debounce version for events.

Reply:
Ah, I could not find where do we set width, but it's in unchanged code.